xbmc/windowing/WindowingFactory.h

```diff
@@ -29,6 +29,12 @@
 #elif defined(TARGET_WINDOWS) && defined(HAS_DX)
 #include "windows/WinSystemWin32DX.h"
 
+#elif defined(TARGET_LINUX)   && defined(HAVE_MIR)   && defined(HAS_GL)
+#include "mir/WinSystemMirGLContext.h"
```

Comment:
Not a code review but please use only one space around operators. Yes, I know... Existing code doesn't follow this request. :wink: